specification/columns/commitmentdiscountunit.md

```diff
@@ -0,0 +1,36 @@
+# Commitment Discount Unit
+
+Commitment Discount Unit relates to [*commitment-based discounts*](#glossary:commitment-based-discount), not to be confused with [*negotiated discounts*](#glossary:negotiated-discount), and is the string value representing the provider-specified measurement unit that corresponds to [CommitmentDiscountQuantity](#commitmentdiscountconsumedquantity) and [CommitmentDiscountPurchasedQuantity](#commitmentdiscountpurchasedquantity).
+
+The CommitmentDiscountUnit column adheres to the following requirements:
+
+* CommitmentDiscountUnit MUST be present in the billing data when the provider supports [*commitment-based discounts*](#glossary:commitment-based-discount).
+* CommitmentDiscountUnit MUST be of type String, and the units of measure used in CommitmentDiscountUnit SHOULD adhere to the values and format requirements specified in the [UnitFormat](#unitformat) attribute.
```

Comment:
This is an interesting question, but since the same clause exists for ConsumedUnit and PricingUnit, I'm going to keep as is until a more holistic approach is offered.